src/main/webapp/frontend/controller/AbstractDetailController.js

```diff
@@ -14,6 +15,34 @@ sap.ui.define([
 		handleFullScreen: function () {
 			this.navToLayoutProperty("/actionButtonsInfo/midColumn/fullScreen");
 		},
+        onAfterCreateOpenDialog: function () {},
+        _openCreateDialog: function (oDialog, sEntityName) {
+          oDialog.open();
+        },
+        pwdOnCancel: function (oEvent) {
+          oEvent.getSource().getParent().close();
+          oEvent.getSource().getParent().destroy();
+        },
+        pwdOnSave: function (oEvent) {
```

Comment:
You might want to add oEvent.getSource().getParent().close(); so that the dialog is closed after pressing "Konnektor jetzt neu starten"